python/lsst/cp/pipe/defects.py

```diff
@@ -170,11 +184,11 @@ class FindDefectsTaskConfig(pexConfig.Config):
     writeAs = pexConfig.ChoiceField(
         doc="Write the output file as ASCII or FITS table",
         dtype=str,
-        default="FITS",
+        default="ECSV",
         allowed={
-            "ASCII": "Write the output as an ASCII file",
+            "ECSV": "Write the output as an ECSV file",
```

Comment:
The api is `writeText` since I deliberately avoided making the underlying text format something people needed to care about.

Reply:
Sounds good. What do you propose to call this in order to let people choose between `.writeText()` and `.writeFits()`? "TEXT"? Or leave it as `"ASCII"`?

Reply:
TEXT or leave ASCII is fine with me.

Reply:
Cool, just dropped that whole commit 👍